docs/index.rst

```diff
@@ -1,4 +1,4 @@
-.. include:: README.rst
+.. include:: ../README.rst
```

Comment:
@pri22296  @sch00lb0y I added them but the toctree is not being generated: yaydoc.herokuapp.com/preview/ujjwalb1996@gmail.com/a3f37342-7bdd-4456-a7ef-de63b198218d_preview/